static/index.js

```diff
@@ -1,5 +1,11 @@
 window.onload = () => {
-    alert("Hello world!");
-  };
+  alert("Hello world!");
```

Comment:
We can remove this now

Reply:
SO how come when I remove alert and put js in there a window doesn't pop up anymore? I though the Window.onload method Creates a pop up windows on "onLoad". Am I wrong?

Reply:
onload does not create a popup window, it simply exists to allow you to run code after the browser has parsed and rendered the entire html/js/css in the initial payload except what's contained in onload. When the browser finishes that render process it then calls onload, no popup, the `alert` function is what generated that popup

Reply:
I understand what onLoad means but was confused about the "window" part. I thought "window" did a popup and then I was telling it to do the popup on "onload". But it sounds like the alert() is what automatically calls a pop up?

Reply:
ah so window is in reference to the tab that you are viewing, it has several functions related to manipulating or understanding the view, for instance `window.innerHeight` and `window.innerWidth` contain the width and height of the rendered page. and alert although accessible on the global scope without calling window, is actually `window.alert`